tests/net/bridge/prj_ip.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_NETWORKING=y
+CONFIG_NET_TEST=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_NET_ETHERNET_BRIDGE=y
+CONFIG_NET_LOG=y
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_NET_PKT_TX_COUNT=10
+CONFIG_NET_PKT_RX_COUNT=10
+CONFIG_NET_BUF_RX_COUNT=10
+CONFIG_NET_BUF_TX_COUNT=10
+CONFIG_ZTEST=y
```

Comment:
I propose not to introduce any new prj.conf file for this but just add a config to testcase.yaml file, see my other comment there.